src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-docker.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Collector for docker containers
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - Docker
+  - Container
+metaDescription: You can collect docker container metrics using the OpenTelemetry collector.
+---
+You can collect metrics about your Docker containers with the OpenTelemetry collector. The collector is a component of OpenTelemetry that collects, processes, and exports telemetry data to New Relic (or any observability backend).
+
+<Callout variant="tip">
+  If you're looking for help with other collector use cases, see the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples) repository.
+</Callout>
+
+Complete the steps below to collect metrics from docker containers:
+
+## Step 1: Sign up for your free account if you haven't already [#signup]
+
+<InlineSignup />
+
+## Step 2: Prerequisites [#prerequisites]
+
+The collector will use the [dockerstats receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/dockerstatsreceiver) component and it only supports Linux.
```

Comment:
Why we support only Linux?
Is this a limitation of the future NRDOT LP?

Reply:
It is not only of the NRDOT LP but also of the [docker stats receiver itself](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/dockerstatsreceiver#readme). This shouldn't be an issue as we have the same limitation for [nri-docker integration](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/linux-installation/container-infrastructure-monitoring/#requirements)

Reply:
Do you think it is worth mentioning the minimum Docker API version supported (v1.22)?
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/dockerstatsreceiver

Reply:
The minimum API version is likely changing, consider [this PR](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21803/files) for instance. Taking it into account, I've just rephrased the requirements paragraph here [48002ac](https://github.com/newrelic/docs-website/pull/13190/commits/48002acb3d09af705201a47889a9f013c3896db2), let me know what do you think.